city_metrix/metrics.py

```diff
@@ -72,7 +70,9 @@ def built_land_with_high_land_surface_temperature(zones):
 
 def urban_open_space(zones):
     built_up_land = EsaWorldCover(land_cover_class=EsaWorldCoverClass.BUILT_UP)
-    open_space = OSMOpenSpace()
+    open_space_tag = {'leisure': ['park', 'nature_reserve', 'common', 'playground', 'pitch', 'track'], 
```

Comment:
same as above, move this to a subclass of OSM

Reply:
Updated OSM layer to have predefined land use types to be reused